readme.txt

```diff
@@ -99,6 +99,7 @@ Please note that our support for the checkout block is still experimental and th
 == Changelog ==
 
 = 3.7.0 - 2022-xx-xx =
+* Add - Disputes list sorting
```

Comment:
Let's rephrase as `Introduce sorting on disputes page.` to align with an entry for filters.

Reply:
Done!